recipes/seaborn/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "0.7.1" %}
+{% set name = "seaborn" %}
+{% set sha256 = "fa274344b1ee72f723bab751c40a5c671801d47a29ee9b5e69fcf63a18ce5c5d" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - numpy
+    - matplotlib
```

Comment:
Why are these 2 needed as build dependencies? Are we linking against them? Do they get imported at build time? Some combination?